phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/common/skin/DetailsListElementSkin.java

```diff
@@ -0,0 +1,79 @@
+package org.phoenicis.javafx.components.common.skin;
+
+import javafx.scene.control.Label;
+import javafx.scene.layout.ColumnConstraints;
+import javafx.scene.layout.GridPane;
+import org.phoenicis.javafx.components.common.control.DetailsListElement;
+import org.phoenicis.javafx.views.common.ColumnConstraintsWithPercentage;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+/**
+ * The skin for the {@link DetailsListElement} component
+ *
+ * @param <E> The concrete type of the element shown in this list element
+ */
+public class DetailsListElementSkin<E> extends SkinBase<DetailsListElement<E>, DetailsListElementSkin<E>> {
+    /**
+     * Constructor
+     *
+     * @param control The control belonging to the skin
+     */
+    public DetailsListElementSkin(DetailsListElement<E> control) {
+        super(control);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public void initialise() {
+        final GridPane container = new GridPane();
+        container.getStyleClass().add("detailsListElement");
+
+        final List<ColumnConstraints> constraints = new ArrayList<>();
+
+        // add the title label
+        final Label title = createTitle();
+
+        constraints.add(new ColumnConstraintsWithPercentage(30));
+
+        container.add(title, 0, 0);
+
+        // TODO: the skin should react to changes done to the additional information list
```

Comment:
Do you want to do this in this PR?

Reply:
No, the `TODO`s are meant as a reminder for the future.
In addition, the additional and detailed information can't change at runtime (at least currently).